db/schema.rb

```diff
@@ -0,0 +1,332 @@
+# This file is auto-generated from the current state of the database. Instead
```

Comment:
Why is the `schema.rb` added? The default db dump is set to sql, I don't think there is a need for this.

Reply:
I did not changed it on purpose, it was just generated automatically.
But the schema is the standard way in rails 6, shouldn't we use that?